tox.ini

```
@@ -208,7 +209,8 @@ deps =
     datastore_pymongo-pymongo03: pymongo<4.0
     datastore_pymysql: PyMySQL<0.11
     datastore_pysolr: pysolr<4.0
-    datastore_redis: redis
+    datastore_redis-redislatest: redis==4.0.2
```

This should be unpinned

```suggestion
    datastore_redis-redislatest: redis
```